docs/app/data/components-page.json

```diff
@@ -682,6 +672,37 @@
             ]
         },
         {
+            "title": "Sliders",
```

Comment:
Yes I see, is that from the jscrollpane in Custom Dropdown? Because that doesn't really count as an input control in my opinion and could be moved to the Buttons section.